*** tests/A-08-06-M-ok.out	2022-10-05 21:24:50.000000000 +0100
--- tests/A-08-06-M-ok.outhyp	2022-10-21 01:57:36.000000000 +0100
***************
*** 17,24 ****
  3 - Operações de terminal
  0 - Sair
  Escolha uma opção: BASIC|969001|cli001|IDLE|0|0
- BASIC|969002|cli002|IDLE|0|0
- BASIC|969003|cli003|IDLE|0|0
  BASIC|999000|cli001|IDLE|0|0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
--- 17,22 ----
