*** tests/A-08-04-M-ok.out	2022-10-20 18:23:38.827866900 +0100
--- tests/A-08-04-M-ok.outhyp	2022-10-21 08:59:14.158555100 +0100
***************
*** 12,19 ****
  3 - Operações de terminal
  0 - Sair
  Escolha uma opção: BASIC|969001|cli001|IDLE|0|0
- BASIC|969002|cli002|IDLE|0|0
- BASIC|969003|cli003|IDLE|0|0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
--- 12,17 ----
