*** testsBoth/A-11-04-M-ok.out	2022-11-01 20:31:13.676676769 +0000
--- testsBoth/A-11-04-M-ok.outhyp	2022-11-02 21:40:31.744820984 +0000
***************
*** 29,35 ****
  Escolha uma opção: 8
  Número do terminal: 969006
  Mensagem de texto a enviar: 1
- O terminal de destino ('969006') está desligado.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 29,34 ----
***************
*** 38,45 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
- 8 - Enviar comunicação de texto
- 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 0
--- 37,42 ----
