*** testsBoth/A-12-05-M-ok.out	2022-10-30 22:57:52.837158700 +0000
--- testsBoth/A-12-05-M-ok.outhyp	2022-11-01 22:13:06.554561000 +0000
***************
*** 30,35 ****
--- 30,36 ----
  Número do terminal: 969008
  Tipo de comunicação a iniciar (VOICE, VIDEO): VOICE
  O terminal de destino ('969008') está em silêncio.
+ O terminal '969001' não pode iniciar comunicações VOICE
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 43,49 ****
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 11
- Não existe nenhuma comunicação em curso.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 44,49 ----
