*** testsBoth/A-12-09-M-ok.out	2022-11-01 20:31:13.676676769 +0000
--- testsBoth/A-12-09-M-ok.outhyp	2022-11-03 15:13:07.293749061 +0000
***************
*** 39,49 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
- 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 11
! VOICE|1|969001|969007|0|0|ONGOING
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 39,48 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 11
! VOICE|0|969007|969007|0|0|ONGOING
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 52,58 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
- 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 0
--- 51,56 ----
