*** testsBoth/A-12-11-M-ok.out	2022-11-01 20:31:13.676676769 +0000
--- testsBoth/A-12-11-M-ok.outhyp	2022-11-03 15:44:49.274494265 +0000
***************
*** 37,47 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
- 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 11
! VIDEO|1|969011|969014|0|0|ONGOING
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 37,46 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 11
! VIDEO|0|969014|969014|0|0|ONGOING
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 50,56 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
- 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 0
--- 49,54 ----
