*** testsBoth/A-16-02-M-ok.out	2022-11-01 20:31:13.680676746 +0000
--- testsBoth/A-16-02-M-ok.outhyp	2022-11-02 00:48:27.642121917 +0000
***************
*** 56,62 ****
  0 - Sair
  Escolha uma opção: 1
  BASIC|969001|cli001|IDLE|0|0
! BASIC|969003|cli001|OFF|0|0|969001
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
  BASIC|969008|cli003|SILENCE|0|0
--- 56,62 ----
  0 - Sair
  Escolha uma opção: 1
  BASIC|969001|cli001|IDLE|0|0
! BASIC|969003|cli001|OFF|0|0|prr.core.BasicTerminal@452b3a41
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
  BASIC|969008|cli003|SILENCE|0|0
