*** testsBoth/A-16-04-M-ok.out	2022-11-01 20:31:13.680676746 +0000
--- testsBoth/A-16-04-M-ok.outhyp	2022-11-02 00:48:27.854119621 +0000
***************
*** 105,112 ****
  3 - Operações de terminal
  0 - Sair
  Escolha uma opção: 1
! BASIC|969001|cli001|IDLE|0|0|969003,969007,969015
! BASIC|969003|cli001|OFF|0|0|969015
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
  BASIC|969008|cli003|SILENCE|0|0
--- 105,112 ----
  3 - Operações de terminal
  0 - Sair
  Escolha uma opção: 1
! BASIC|969001|cli001|IDLE|0|0|prr.core.BasicTerminal@24d46ca6,prr.core.BasicTerminal@4517d9a3,prr.core.FancyTerminal@372f7a8d
! BASIC|969003|cli001|OFF|0|0|prr.core.FancyTerminal@372f7a8d
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
  BASIC|969008|cli003|SILENCE|0|0
